.gitignore

```diff
@@ -1,6 +1,12 @@
 # Ignore everything
 /*
 
+# Ignore eclipse files
```

Comment:
Why is this needed? we use an gitinclude here? (see first line).

If this is for subproject stuff see how its done for the other files such as .gradle bellow

Reply:
I don't know what gitinclude is, but every time I work on the project I have hundreds of unchecked files from the sub-modules because they're not excluded.

Reply:
Added an image to illustrate what this PR fixes.

Reply:
This isnt the correct fix, look at the subproject section that already exists.

Reply:
I see this:

```
# Subprojects
!/fabric-*
/fabric-*/.gradle
/fabric-*/build
```

But that doesn't correctly exclude all your files. If you're going to do that you have to do:
```
# Subprojects
!/fabric-*/src
!/fabric-*/build.gradle
/fabric-*/.gradle
/fabric-*/build
```

Reply:
Actually, no. That still doesn't include the `src` folder.

Reply:
You should just exclude the files you don't want appearing in the repo. Including them individually is exactly what leads to these kinds of problems.